kalmantv/eigen/KalmanTV_raw.h

```diff
@@ -0,0 +1,289 @@
+/// @file KalmanTV_raw.h
+
+#ifndef KalmanTV_RAW_h
+#define KalmanTV_RAW_h 1
+
+// #undef NDEBUG
+// #define EIGEN_RUNTIME_NO_MALLOC
+#include <Eigen/Dense>
+#include <iostream>
+#include "KalmanTV.h"
+
+namespace kalmantv_raw {
```

Comment:
I think you can just put this in the `kalmantv` namespace, instead of creating a different namespace...